src/fact_base.py

```diff
@@ -86,3 +96,13 @@ def do_self_test():
                 'Please run `alembic upgrade head` from `src` to update the schema.'
             )
             raise DbInterfaceError('Schema mismatch')
+
+    @staticmethod
```

Comment:
I personally don't think private static methods make much sense.
Why not make it a function in in the module?

Reply:
converted the class method to a function